<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZCMonitorVOBC</class>
 <widget class="QWidget" name="ZCMonitorVOBC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QDateTimeEdit" name="endDateEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDateTimeEdit" name="startDateEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="queryButton">
        <property name="text">
         <string>开始检测</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>选择消息包</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="filter_cbo">
        <item>
         <property name="text">
          <string>列车控制信息</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>应用层注册/注销响应</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>应用层注销请求</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>特殊控制报文</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="8">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="nextButton">
        <property name="text">
         <string>后一页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>转到第</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="switchPageLineEdit"/>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="switchPageButton">
        <property name="text">
         <string>转到</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="prevButton">
        <property name="text">
         <string>前一页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="totalPageLabel">
        <property name="text">
         <string>共   0 条记录</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="currentPageLabel">
        <property name="text">
         <string>共   0  页  当前第 0 页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="10">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>每页显示100条</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>每页显示500条</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>每页显示1000条</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>每页显示2000条</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
